test/unit/minify-sourcemap-register/output.js

```diff
@@ -1,2 +1 @@
 require('./sourcemap-register.js');(()=>{if(typeof __nccwpck_require__!=="undefined")__nccwpck_require__.ab=__dirname+"/";var e={};(()=>{var _=e;const o="qux";console.log?.("hello");_.foobar=o})();module.exports=e})();
-//# sourceMappingURL=index.js.map
```

Comment:
This looks like a bug since the source map comment is missing

Reply:
Seems to be a limitation of swc as it is impossible to pass an object to the `minify.sourceMap` option.

https://github.com/vercel/ncc/blob/36a7568175b5938353f3171b17bda68ebe025d06/src/index.js#L485-L489

cc @kdy1

Reply:
Should SWC write `//# sourceMappingURL=index.js.map`?

https://github.com/swc-project/swc/blob/a53c60db18bd90539f3449348cbb166f32f46223/crates/swc_compiler_base/src/lib.rs#L259-L271

SWC has relevant code for sourceMap = `inline` but not for `true`. Does terser emit one?

Reply:
https://terser.org/docs/api-reference/#source-map-options so there was an option. I'll implement this option today or tomorrow and publish a new version of `@swc/core`

Reply:
https://github.com/swc-project/swc/pull/10346 implements the option, and I'll publish a new version later today, after merging some PRs.

Sorry for the late response! The mention email was buried due to lots of other emails. (Mostly PR review requests)

Reply:
Sure. Lemme see if the tests passed here after bumping the swc to the latest version later today.

Reply:
A new version is published! Can you try updating `@swc/core`?

Reply:
@kdy1 Just bump the swc to the latest version, and it appears to be working, thanks for fixing this! cc @styfle 

See also CI test runs: https://github.com/SukkaW/ncc/pull/2